src/main/resources/application.conf

```diff
@@ -33,6 +33,11 @@ docker {
   // dockerToken = ""
 }
 
+jobavoidance {
+  // Set to true in order to avoid running previously run jobs.
+  enabled = false
+}
+
```

Comment:
Ah I see what you guys mean. Agree with Khalid as JA was requested to be something very specifically enameled